Templates/GenericWebHook-CSharp/function.json

```diff
@@ -5,6 +5,7 @@
             "type": "httpTrigger",
             "direction": "in",
             "webHookType": "genericJson",
+            "authLevel": "function",
```

Comment:
authLevel only applies to non WebHook functions. WebHooks by definition use their own auth. So I don't think you should show this authLevel for WebHook functions, and you shouldn't have this default value in there. See controller auth code [here](https://github.com/Azure/azure-webjobs-sdk-script/blob/master/src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs#L60).